src/App.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import RecipeApp from "./RecipeApp";
+
+export default function App() {
```

Comment:
Why the choice of returning just a single component, and not add RecipeApp content directly here?

Reply:
No strong reason :shrug: 
I just intended to leave the create-react-app stuff as-is and take the actual app to its own folder.